utils/appliance/__init__.py

```diff
@@ -1604,7 +1604,7 @@ def start_evm_service(self, log_callback=None):
         self._evm_service_command('start', expected_exit_code=0, log_callback=log_callback)
 
     @logger_wrap("Restart EVM Service: {}")
-    def restart_evm_service(self, rude=False, log_callback=None):
+    def restart_evm_service(self, rude=False, wait_for_web_ui=False, log_callback=None):
```

Comment:
If I did set rude to true in a previous version, that would have indeed been an error.  @mshriver and I went over it again and it looks correct now and that's how I intended it to read.  The wording of the comment says I should have left the default to true, so if I did set rude=True then that comment is inconsistent.